File: LeetCode/0014. Longest Common Prefix/JavaScript/01. Longest Common Prefix.js
var longestCommonPrefix = function(strs) {
    strs.sort();
    let firstWord = strs[0];
    let secondWord = strs[strs.length - 1];
    let prefixLength = 0;
    while(prefixLength < firstWord.length && prefixLength < secondWord.length){
        if(firstWord[prefixLength] === secondWord[prefixLength]){
            prefixLength++;
        }
        else{
            break;
        }
    }
    return firstWord.substring(0, prefixLength);
};

File: LeetCode/0014. Longest Common Prefix/C#/01. Longest Common Prefix.cs
string LongestCommonPrefix(string[] strs)
{
    Array.Sort(strs);
    string firstWord = strs[0];
    string secondWord = strs[strs.Length - 1];
    int prefixLength = 0;
    while (prefixLength < firstWord.Length && prefixLength < secondWord.Length)
    {
        if (firstWord[prefixLength] == secondWord[prefixLength])
            prefixLength++;
        else
            break;
    }
    return firstWord.Substring(0, prefixLength);
}